Example-Audit-REXX/RXAUDIT1.rexx

```diff
@@ -0,0 +1,2478 @@
+/*        -D <data set name> is the name of a pre-existing partitioned */
+/*           data set, if the LPAR audit is successful, a member       */
+/*           containing the audit information in CSV format will be    */
+/*           stored into the data set, the member will either be named */
+/*           LOCAL or the CPC name provided via -C option              */
+/*     Optional input parameters:                                      */
+/*        -C <CPCname> name of the CPC to audit, default is LOCAL CPC  */
+/*        -I indicate running out of ISV REXX environment,             */
+/*           default is TSO/E                                          */
+/*        -V turn on additional verbose JSON tracing                   */
+/*                                                                     */
+/*        EX 'HWI.HWIREST.REXX(RXAUDIT1))'                             */
+/*            '-D HWI.AUDIT -I'                                        */
+/*            data set is HWI.AUDIT, run out of ISV REXX,              */
+/*            if successful will create HWI.AUDIT(LOCAL)               */
+/*                                                                     */
+/* DEPENDENCIES                                                        */
+/*     none.                                                           */
+/*                                                                     */
+/*    NOTES:                                                           */
+/* No recovery logic has been supplied in this sample.                 */
+/*                                                                     */
+/*    REFERENCE:                                                       */
+/*        See the z/OS MVS Programming: Callable Services for          */
+/*        High-Level Languages publication for more information        */
+/*        regarding the usage of HWIREST and JSON Parser APIs.         */
+/*                                                                     */
+/*                                                                     */
+/* END OF SPECIFICATIONS  * * * * * * * * * * * * * * * * * * * * * * **/
+
+MACLIB_DATASET = 'SYS1.MACLIB'
+
+TEMP_DATASET = '' /* required input via -D */
+MEMBER_NAME = 'LOCAL' /* default CPC */
+
+TRUE = 1
+FALSE = 0
+
+bRC = 0          /* HWIHOST rc */
+HWIHOST_ON = FALSE
+PARSER_INIT = FALSE
+ISVREXX = FALSE  /* default to TSO/E, enable via -I */
+VERBOSE = FALSE  /* JSON parser specific, enabled via -V */
+localCPC = TRUE  /* default to lpars on the LOCAL CPC, change via -C */
+
+/* Complex Properties constants */
+storageCentralAllocProp = 'storage-central-allocation'
+storageExpandedAllocProp = 'storage-expanded-allocation'
+
+/*********************************/
+/* Get program args              */
+/*********************************/
+parse arg argString
+if GetArgs(argString) <> 0 then
+   exit -1
+
+/*********************************/
```

Comment:
Might not be necessary for this review, but maybe you'll want to change the flower box comments to be in this format:
`/*********************************
 * Before going  much further,
 * ensure the data set specified
 * actually exists               
 *********************************/`